build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -1,9 +1,13 @@
 # Provide the flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp.
 # If no L4 transport proto is given, we consider tcp as default.
-flowCollectorAddr: ""
+externalFlowCollectorAddr: ""
```

Comment:
We comment out the default values in the yamls. Would you follow that convention?